rfhub2/db/model/mixins.py

```diff
@@ -19,3 +19,10 @@ def arg_string(self) -> str:
         Old implementation saves args list as JSON in text field, this is more readable representation for UI
         """
         return ", ".join(json.loads(self.args)) if self.args else ""
+
+    @property
```

Comment:
IMO this can be removed from API and put  in UI if it will be used after all (chips?)

Reply:
UI? How is it possible? I don't see that in ui folder